SelectionShort Java/SelectionShort.java

```diff
@@ -0,0 +1,28 @@
+
+package SelectionShort;
+
+public class SelectionShort {
+    public static void main(String[] args) {
+        
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpw1m34vy4/SelectionShort Java/SelectionShort.java
+++ b/tmp/tmpw1m34vy4/SelectionShort Java/SelectionShort.java
@@ -3,7 +3,7 @@
 
 public class SelectionShort {
     public static void main(String[] args) {
-        
+
         int [] number = {8,3,4,7,2,6,5,1,10,9};
         Selection_Short(number);
     }
```